lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -103,6 +103,11 @@ const ziti_tunnel_ctrl* ziti_tunnel_init_cmd(uv_loop_t *loop, tunneler_context t
 
 IMPL_ENUM(mfa_status, MFA_STATUS)
 
+void ziti_set_refresh_interval(unsigned long seconds) {
+    unsigned long old = refresh_interval;
```

Comment:
is there any reason to keep the old value?

Reply:
no, I was originally returning the old value, but it has no real use.
I'll clean up